components/style/properties.mako.rs

```diff
@@ -16,7 +16,7 @@ use std::mem;
 use std::sync::Arc;
 
 use app_units::Au;
-use cssparser::{Parser, Color, RGBA, AtRuleParser, DeclarationParser, Delimiter,
+use cssparser::{Parser, RGBA, AtRuleParser, DeclarationParser, Delimiter,
```

Comment:
This is just me trying to work around name clashes; it may be that removing this causes the generated code to be completely wrong (meaning, that references to `cssparser::Color` might get incorrectly replaced by our local `Color` struct...). So this may very well be one of the parts that should get reverted.